### PR TITLE
Add Grafana dashboard for Plus

### PR DESCRIPTION
**Pull request by LorcanMcVeigh**
### Proposed changes
Add a Grafana dashboard for plus metrics.

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto master
- [x] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

**Review by pleshakov — CHANGES_REQUESTED** [review]
Hi @LorcanMcVeigh 

The PR doesn't add the docs unfortunately. You can take a look at https://deploy-preview-1194--nginx-kubernetes-ingress.netlify.app/  -- there are no docs there.

I like the overall structure and content. However, there are a few problems. Please see my suggestion.

Additionally:
* I suggest moving this folder to the root repo folder, similar to this https://github.com/nginxinc/nginx-prometheus-exporter/tree/master/grafana . So it will have both the dashboard and the README. Having the dashboard and readme there will also allow people to easily use the master version of the dashboard without waiting for a new release.
* In the docs, I suggest adding adding a Graphana section here https://deploy-preview-1194--nginx-kubernetes-ingress.netlify.app/nginx-ingress-controller/logging-and-monitoring/prometheus/  that will reference the dashboard:
 
## Enabling Metrics
. . .
## Available Metrics
. . .
## Graphana Dashboard 

You can visualize the Prometheus metrics using the Graphana dashboard from our [GitHub repo](). The dashboard is only available for NGINX Plus.

File: docs-web/logging-and-monitoring/grafana/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+![dashboard](./dashboard.png)
+
+## Graphs
+
+The dashboard comes with 4 rows with the following graphs for NGINX metrics:
+
+* Environment Variables
+    * Nginx Plus Reload (`nginx_last_reload_status`). This graph shows the state of the last Nginx Plus reload, `Successful`/`Failed`.
+    * Last Reload Time (`nginx_last_reload_milliseconds`)graph, shows duration of the last reload in milliseconds.
+    * Reloads ( `nginx_reloads_total`). This graph shows the total times Nginx has reloaded.
```
Comment (Dean-Coakley) [inline docs-web/logging-and-monitoring/grafana/README.md:31]:
```suggestion
    * Reloads (`nginx_reloads_total`). This graph shows the total times Nginx has reloaded.
```

File: docs-web/logging-and-monitoring/grafana/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    * Reload Errors (`nginx_reload_errors_total`) graph shows the total number of times Nginx has failed to reload.
+    * Netowrk I/O (`nginxplus_server_zone_received` and `nginxplus_server_zone_sent`). This graphs shows the traffic sent and recieved by the Ingress Controller.
```
Comment (Dean-Coakley) [inline docs-web/logging-and-monitoring/grafana/README.md:33]:
```suggestion
    * Network I/O (`nginxplus_server_zone_received` and `nginxplus_server_zone_sent`). This graphs shows the traffic sent and received by the Ingress Controller.
```

File: docs-web/logging-and-monitoring/grafana/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+* Ingress Metrics
+    * HTTP Request Volume (`nginxplus_http_requests_total`). This graphs shows the total number of HTTP requests recieved by the Ingress Controller.
```
Comment (Dean-Coakley) [inline docs-web/logging-and-monitoring/grafana/README.md:35]:
```suggestion
    * HTTP Request Volume (`nginxplus_http_requests_total`). This graphs shows the total number of HTTP requests received by the Ingress Controller.
```

File: docs-web/logging-and-monitoring/grafana/NGINXPlusICDashboard.json [hunk 4]
```diff
@@ -0,0 +1,1924 @@
...
+              "metric": "container_memory_usage:sort_desc",
+              "refId": "C",
+              "step": 10
+            }
+          ],
+          "thresholds": [],
+          "timeFrom": null,
+          "timeRegions": [],
+          "timeShift": null,
+          "title": "Sucess Rates Over Time",
```
Comment (Dean-Coakley) [inline docs-web/logging-and-monitoring/grafana/NGINXPlusICDashboard.json:862]:
```suggestion
          "title": "Success Rates Over Time",
```

File: docs-web/logging-and-monitoring/grafana/NGINXPlusICDashboard.json [hunk 5]
```diff
@@ -0,0 +1,1924 @@
...
+      "aliasColors": {
+        "Received": "rgb(0, 150, 57)",
+        "Sent": "rgb(255, 255, 255)"
+      },
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "Prometheus",
+      "decimals": 2,
+      "description": "Bytes received from clients and  sent to clients.",
```
Comment (Dean-Coakley) [inline docs-web/logging-and-monitoring/grafana/NGINXPlusICDashboard.json:299]:
```suggestion
      "description": "Bytes received from clients and sent to clients.",
```

**Review by martialonline — APPROVED** [review]
👍

**Review by vepatel — APPROVED** [review]

**Pull request merged**